cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/InputsUtil.java

```diff
@@ -368,6 +375,16 @@ public static void validateKeyOrValueString(String input, boolean isKey) {
         }
     }
 
+    public static String getValidArn(String input) {
+        Pattern pattern = Pattern.compile("[a-zA-Z0-9\\:\\-\\/]+");
+        if (!input.contains(COLON) || !input.startsWith(ARN) || !input.contains(AWS) || !input.contains(LOAD_BALANCING_API)
```

Comment:
This if is hard to follow, also !input.contains(COLON) seems to be redundant IMHO because of countMatches(input, COLON) != 5. Also, can't we specify an ARN as arn:partition:service:region:account-id:resourcetype:resource (this has 6 colons)? My suggestion is to let the validation to the Amazon service.